templates/user/profile.html

```diff
@@ -79,6 +80,15 @@ <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
   })
   localizeField.addEventListener('change', event => event.target.checked && localize())
 
+  const publishField = document.querySelector('[name=publish]')
```

Comment:
but don't succeed to fix it ... in each case the ID block me because I want a string without number or sign